google/cloud/language/v1beta1/language_gapic.yaml

```diff
@@ -3,7 +3,7 @@ language_settings:
   java:
     package_name: com.google.cloud.language.spi.v1beta1
   python:
-    package_name: google.cloud.language.v1beta1
```

Comment:
We discussed offline and decided to use `gapic` rather than `gax`. The former means Google API Codegen, and is a more appropriate name than the latter, which mean Google API eXtensions (which specifically refers to [gax-python](https://github.com/googleapis/gax-python)).

Also, we will soon change the Python package names on PyPI from `gax-google-*` to `gapic-google-*` (not part of this PR). It's much easier to make this change now than it is post-1.0.

FYI @tseaver @dhermes @daspecster @jgeewax